pkg/certificate/manager_test.go

```diff
@@ -9,11 +9,11 @@ import (
 	"github.com/cskr/pubsub"
 	tassert "github.com/stretchr/testify/assert"
 	trequire "github.com/stretchr/testify/require"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 
-	"github.com/openservicemesh/osm/pkg/apis/config/v1alpha2"
 	"github.com/openservicemesh/osm/pkg/certificate/pem"
 	"github.com/openservicemesh/osm/pkg/constants"
+	configFake "github.com/openservicemesh/osm/pkg/gen/client/config/clientset/versioned/fake"
```

Comment:
uppercase

Reply:
If this is non-blocking, I'll resolve this in a follow-up PR. There are multiple instances of this casing in aliased package names that could be cleaned up in a single PR. If we want to enforce this pattern we could add it to our linter.